python/cuda_cccl/setup.py

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import shutil
+from pathlib import Path
+
+from setuptools import setup
+
+PROJECT_PATH = Path(__file__).resolve().parent
+CCCL_PATH = PROJECT_PATH.parents[1]
+
+
+def copy_cccl_headers_to_cuda_cccl_include():
```

Comment:
I don't think this is the best way to solve this problem. It probably works in many cases, but it also has a lot of sharp edges. The better solution for this kind of installation (the problem described in https://github.com/NVIDIA/cccl/blob/d6253b5ec9f8011175d9d13c43e0f7d5f86335ff/ci/test_python.sh#L23) is to override setuptools's commands. Here is an example: https://stackoverflow.com/questions/2145779/setup-py-installing-just-a-pth-file/71137790#71137790

Reply:
Done: 71fd243baab28a84739406e05927854e9a863f1f